test/utils.js

```diff
@@ -42,3 +43,50 @@ tap.test('createUndiciRequestOptions', t => {
     t.strictDeepEqual(found, wanted)
   })
 })
+
+tap.test('sort1d', t => {
+  t.plan(1)
```

Comment:
Add another test that uses the existing `.sort` to prove they have the same output. You can even use my existing solution if you need.

Reply:
Good idea!